src/open_inwoner/components/templates/components/File/File.html

```diff
@@ -7,7 +7,14 @@
     {% else %}
         {% icon icon="description" outlined=True %}
     {% endif %}
+
```

Comment:
here you can close the new file__symbol paragraph, and in File.css add this: 
`.file__symbol {
    display: flex;
    flex-direction: column;
    align-items: center;
    background-color: var(--color-gray-lightest);
    padding: var(--spacing-large) 0;
    margin: 0;
    box-sizing: content-box;

    *[class*='icon'],
    *[class*='Icon'] {
      align-self: stretch;
      padding: var(--spacing-large);
    }
  }`
 
**NB:** This means that this can be **removed** there: 
 
`&__file > *[class*='icon']:first-child,
  &__file > *[class*='Icon']:first-child {
    background-color: var(--color-gray-lightest);
    padding: var(--spacing-large);
    box-sizing: content-box;
  }`